examples/layers.py

```diff
@@ -0,0 +1,42 @@
+"""
+Display multiple image layers using the add_image API and then reorder them
+using the layers swap method and remove one
+"""
+
+import sys
+from PyQt5.QtWidgets import QApplication
+
+from skimage import data
+from skimage.color import rgb2gray
+from napari import Window, Viewer
+
+
+# starting
+application = QApplication(sys.argv)
+
+# create the viewer and window
+viewer = Viewer()
+window = Window(viewer)
+# add the first image
+viewer.add_image(rgb2gray(data.astronaut()))
+viewer.layers[-1].name = 'astronaut'
```

Comment:
you can pass in the `name` property in `add_image`

Reply:
done